src/test/java/com/glovoapp/planningpoker/e2e/pages/MainPage.java

```diff
@@ -0,0 +1,62 @@
+package com.glovoapp.planningpoker.e2e.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public final class MainPage {
+
+    private static final String url = "https://glovo-planning-poker.herokuapp.com/";
```

Comment:
Constants names should be `UPPER_CASE`:

```suggestion
    private static final String URL = "https://glovo-planning-poker.herokuapp.com/";
```